eng/pipelines/client.yml

```diff
@@ -89,9 +89,9 @@ jobs:
           goals: 'install'
 
       - task: Maven@3
-        displayName: 'Install all client libraries (for SpotBugs analysis)'
+        displayName: 'Install Module'
```

Comment:
nit: I'd leave the "(for SpotBugs analysis)" in the display name

Reply:
Added `(for SpotBugs analysis)` back to the name.